packages/atomic/src/components/atomic-external/atomic-external.tsx

```diff
@@ -0,0 +1,35 @@
+import {Component, Prop, Listen} from '@stencil/core';
```

Comment:
We have a `samples/atomic` package in the repo.

Should we not use that anymore/remove it, or instead put all such code samples in there ?

Reply:
I believe the intent of this PR is to provide `atomic-external` as a core component that devs can use to slot other components into,

Reply:
This file is a component, but for the html files it's meant to display how the feature (custom & external component) works and it can be reused for testing.

Reply:
I think the samples/atomic is for docs, not sure, we can remove it IMO

cc @dlutzCoveo

Reply:
@ThibodeauJF @olamothe Yes! `samples/atomic` can be removed or repurposed.